app/models/mongoidable/ability.rb

```diff
@@ -96,14 +96,21 @@ def method_missing(name, *args, &block)
     end
 
     def valid_for_parent?
-      true
+      self.class.valid_for?(_parent)
```

Comment:
```suggestion
      self.class.valid_for?(_parent.class)
```

Reply:
Also, since we didn't see this till review, it would be cool if there were a test that covered it...